tools/functional-tester/README.md

```diff
@@ -14,4 +14,13 @@ etcd agent is a daemon on each machines. It can start, stop, restart, isolate an
 
 ## etcd tester
 
-etcd functional tester control the progress of the functional tests. It calls the the RPC of the etcd agent to simulate various test cases. For example, it can start a three members cluster by sending three start RPC calls to three different etcd agents. It can make one of the member failed by sending stop RPC call to one etcd agent.
\ No newline at end of file
+etcd functional tester control the progress of the functional tests. It calls the the RPC of the etcd agent to simulate various test cases. For example, it can start a three members cluster by sending three start RPC calls to three different etcd agents. It can make one of the member failed by sending stop RPC call to one etcd agent.
+
+## with Docker
+
```

Comment:
The script <bla> can be used to run the functional tests in a confined environment using Docker Compose. Running the script requires:
- Docker 1.9+ (with networking support)
- `docker-compose`
- A multi-arch Go toolchain (if on OS X)